app/overrides/add_activation_keys_input.rb

```diff
@@ -15,7 +15,7 @@
 
 Deface::Override.new(:virtual_path => "hosts/_form",
                      :name => "hosts_update_environments_select",
-                     :insert_before => 'erb[loud]:contains("host_puppet_environment_field")',
+                     :insert_after => 'erb[loud]:contains("Deploy on")',
```

Comment:
Is this using the label?  And if so wouldn't this break with i18n on non-english languages?